### ISSUE TITLE
Client not running after hibernation

### ISSUE THREAD

**Issue by MarcSN311** [nextcloud/desktop#1777]
This happend to me on my Notebook for some time now. After Hibernation the nextcloud client has to be manually started, the process iss not running anymore. I thought it was related to my Setup so I did not open an Issue, but last week I installed a new computer and it also happens on this one.

### Expected behaviour
After Hibernation Client should still be running.

### Actual behaviour
Client process is not running.

### Steps to reproduce
1. Install a Windows 10 Computer
2. Install Nextcloud Client
3. Hibernate

### Client configuration
Client version: 2.6.2stable-Win64 (build 20191224). (Also happend with earlier versions)

Operating system: Windows 10 x64 (Enterprise Insider Build; Pro 1909)

OS language: German

Installation path of client: C:\Program Files\Nextcloud

### Logs
Nothing in the logs.

**Comment by ostasevych** [comment]
The same with 2.7.0beta1, x64. The client is released from the memory after Win10 laptop waking up.

**Comment by jameskimmel** [comment]
If have the same problem under Windows 10 with Version 3.0.1.

It is very annoying and should be very high on the priority list IMO! If offer nextcloud to non tech savvy friends and parents and it is hard to explain to them, that they should pause and resume every time they open up their laptops.

**Comment by jameskimmel** [comment]
#489 is the same issue.

**Comment by er-vin** [comment]
> #489 is the same issue.

Actually I don't think so, in #489 the client is still running on resume, here what seems to happen is that it quits completely or crashes. Same trigger perhaps but two very different symptoms.

**Comment by jameskimmel** [comment]
> 
> 
> > #489 is the same issue.
> 
> Actually I don't think so, in #489 the client is still running on resume, here what seems to happen is that it quits completely or crashes. Same trigger perhaps but two very different symptoms.

Ohh yes you are right. Sorry, I just assumed it to be the same trigger. But you are right, these are different symptoms.

**Comment by renky** [comment]
> If have the same problem under Windows 10 with Version 3.0.1.
> 
> It is very annoying and should be very high on the priority list IMO! If offer nextcloud to non tech savvy friends and parents and it is hard to explain to them, that they should pause and resume every time they open up their laptops.

I can confirm it with Client 3.1.1 on Windows 10. 
The interesting thing is: I recently reinstalled my whole system due to new Mainboard and CPU and I also used the Client 3.1.1 on my system before (also Windows 10) where I didn't have this issue!! so it doesn't seem to be a general problem, but I cannot find a difference that might be the reaons... 
On my system before I ran a Windows 10 that was upgraded from Windows 7. On the other Hand I already had Nextcloud Clients 2.x on the old system and only reinstalled the 3.x on it... I also had Hibernation option activated and used to use it nearly everyday... after resuming the system nextcloud still ran and synced... On the new system (Windows 10 from the beginning, NC-Client 3.1.1 from the beginning) I ALWAYS have the issue that it says "Network access error" after resuming the system. I have do stop NC-Client and restart it manually, then it syncs immediatelly...
